### PR TITLE
LRAC-14587 follow up unit tests

### PR DESCRIPTION
**Pull request by fmarcio**
Jira issue https://liferay.atlassian.net/browse/LRAC-14587

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

**Review by interaminense — APPROVED** [review]
LGTM!

**Comment by interaminense** [comment]
ci:forward

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/View.jsx [hunk 1]
```diff
@@ -128,6 +128,10 @@ describe('View Channel', () => {
 			'show'
 		);
 
+		expect(
+			screen.getByText('Unable to Delete Property')
+		).toBeInTheDocument();
```
Comment (ViniLopes20) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/View.jsx:133]: Hey @fmarcio, is it possible to validate that the modal text is present and not just the modal title?

See [here](https://liferay.atlassian.net/wiki/spaces/ENGAC/pages/2456355170/Test+Plan+LRAC-14082+Enable+Property+Deletion+from+DXP#Check-error-modal-message-and-hyperlink-on-deleting-property-that-has-sites-or-channels-synced---P3)

**Comment by ViniLopes20** [comment]
ci:stop

**Comment by interaminense** [comment]
Resent https://github.com/liferay-ac/liferay-portal/pull/732

**Pull request closed without merging**